### PR TITLE
fixed checkstyle errors and added logic to methods

### PR DESCRIPTION
**Pull request by Jacalius**

### CODE REVIEW

File: src/main/java/mate/academy/service/UserService.java [hunk 1]
```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```
Comment (Rommelua) [inline src/main/java/mate/academy/service/UserService.java:17]: do not use static methods

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -11,6 +13,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = UserService.findByEmail(email);
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```
Comment (Rommelua) [inline src/main/java/mate/academy/service/AuthenticationService.java:20]:
```suggestion
        return user != null && user.getPassword().equals(password));
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/UserService.java (hunk 1 above)
Comment (Jacalius) [reply src/main/java/mate/academy/service/UserService.java:17]: If I don't use the static User method, I'll get an error during compilation (java: non-static method findByEmail(java.lang.String) cannot be referenced from a static context)

Comment (MentorMA) [reply src/main/java/mate/academy/service/UserService.java:17]: What approach do you need for invoke non-static method?

Comment (MentorMA) [reply src/main/java/mate/academy/service/UserService.java:17]: don`t make this method static

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/UserService.java (hunk 1 above)
Comment (MentorMA) [reply src/main/java/mate/academy/service/UserService.java:17]: If you can`t resolve this problem, feel free ask in chat, mentors not answer in git

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user1 = new UserService();
```
Comment (okuzan) [inline src/main/java/mate/academy/service/AuthenticationService.java:16]: make this service a class level field, use a better name, like userService, naming with numbers is a bad practice

**Review by okuzan — CHANGES_REQUESTED** [review]
one issue left

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 4]
```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
+    public UserService getUserService() {
+        return userService;
+    }
+
```
Comment (okuzan) [inline src/main/java/mate/academy/service/AuthenticationService.java:20]: redundant

**Review by okuzan — APPROVED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 4 above)
Comment (Jacalius) [reply src/main/java/mate/academy/service/AuthenticationService.java]: removed redundant

**Pull request closed without merging**